src/Tribe/PUE/Checker.php

```diff
@@ -416,6 +416,7 @@ public function set_key_status( $valid ): void {
 		 * Also, other hooks related to the automatic updates (such as checking against API and what not (@from Darren)
 		 *
 		 * @since 6.5.1 Added `initialize_license_check` action.
+		 * @since TBD Moved `monitor_active_plugins` and `initialize_license_check` to `setup_pue_license_hooks`, and run on `tribe_common_loaded`.
```

Comment:
```suggestion
		 * @since TBD Moved `monitor_active_plugins` and `initialize_license_check` to `setup_pue_license_hooks`, and run on `admin_init`.
```